### DISCUSSION TITLE
The Senate as a functional Building

### DISCUSSION THREAD

**Discussion by Mephisto-Fibs** [Keriew/augustus#425] [Ideas]
I have an idea for a new and functional concept for the Senate and I would like to ask you what you think.

There are two tiers of the Senate, like bevore, but the Senate would not expand through high areal desirability but through the existence of patricians in the city. 

The _first tier_ of the Senate governs basic policies that, so far, were togglable through the main menu in the Augustus game options:

1: allow merging of houses 
2: allow global labor pool 

I suggest, as soon as the Senate has been built and has a full workforce, these options can be activated by klicking a button in the senate info window to take effect in the city

The _second tier_ of the Senate becomes activated as soon as the first patrician appears in the city. Then it unlocks one of the following two effects that can be chosen: 

- First Option: The wealthy patrician families in the city pay their tribute to Caesar. Caesar likes the attention of your wealthy citizens so much that his favor towards you grows by 5 to 10 points annually. But, as you are a patrician as well, your personal pay is halved as long as this policy is active. 

- Second Option: The patricians want to increase their influence in the city and lobby the city governor through money gifts (increase of personal funds for x amount). But the patricians bind their monetary gifts to conditions, which have to be met first.... It could look like this: 'Build a (large temple, small/big mausoleum, ...)' in the city and you will get X-amount of money for your personal funds. 

Any further ideas by you guys? additions? criticism?

**Comment by Purakane** [comment]
The senate building is pretty useless at this point and is mostly used to improve disability than anything else.

**Reply by dmyersturnbull** [reply]
*For posterity:* Also mentioned in #289: https://github.com/Keriew/augustus/discussions/289#discussioncomment-2249750.

Personal opinion, I LOVE the idea of making the senate a useful building, and adding city control options or researchers sounds like a good way to do that. I'm not sold on global labor pool because it's too large a change: with it off, you design your city quite differently. (Perhaps if the unemployed citizen walk algorithm is improved so that it can find, e.g., a farm 2 blocks away, then toggling global labor pool from the Senate building would be less significant.)

**Comment by tmancini85** [comment]
Senate historically expressed patrician power: its absence should reduces patrician political influence in the city, a condition which they should not appreciate, of course. It's possible to try to simulate this interactions through sentiment (a system which has been enhanced a lot in Augustus) and taxes. The governor should exercise his power and oppose patrician one through governor's residence(s) influence.

In the Caesar 3 inner help it is said the governor's residence is going to be the "centre" of the more eminent neighbourhood, and the historical supplementary analysis states that the residence has to be more sumptuous than rich people houses, to show governor's superiority over them, but not too sumptuous, to avoid resentment and rebellion from poor persons. I tried to take into account these never implemented intentions in my reasoning.

I read the Sentiment & Crime chapter in the Version 4.0 manual, but it's hard for me to select adequate sentiment factor numerical values, so I expressed them in relative terms.

I tried to evaluate possible scenarios, and, if my mechanism can be exploitable, effects on sentiment and taxes should be adjusted to obtain the desired bonus/malus balance.

**No Senate**:
- sentiment (negative sentiment factor values as follows):
    - patrician villas/palaces (this represents patrician aversion to Senate absence):
        - 4x4: very high;
        - 3x3: high;
        - 2x2: medium;
    - plebeian houses (this represents the economic and clientelistic power of the patricians, who can instigate plebeians against the governor):
        - visited by a patrician from a 4x4 villa/palace: high or very high;
        - visited by a patrician from a 3x3 villa/palace: medium or high;
        - visited by a patrician from a 2x2 villa/palace: small or medium;
        - not visited by a patrician: very small or small;
- politics:
    - each patrician villa/palace pays 90% of the potential owed taxes (with no Senate, it's harder, but not impossible, for patricians to gather and to pressure the governor to grant favourable conditions).

**Senate built**:
- sentiment (sentiment factor values as follows):
    - patrician villas/palaces (this represents patrician desire to be in the more eminent neighbourhood):
        -  not visited by a Senate tax collector: very small or small negative;
        - upgraded Senate: very small or small positive;
    - plebeian houses (to integrate, and in additional to, normal Housing inequalities factor):
        - visited by a patrician from a 4x4 villa/palace: medium negative;
        - visited by a patrician from a 3x3 villa/palace: small negative;
        - visited by a patrician from a 2x2 villa/palace: very small negative;
- politics:
    - each patrician villa/palace pays only 65%  of the potential owed taxes (patricians can gather in the Senate and issue favourable laws to them);

**Governor's residence(s) built**:
- it/they should need labourers (let's say 10, 18 or 25 persons based on size):
    - it/they will spawn a governmental official;
- sentiment (positive or negative sentiment factor values as follows):
    - patrician villas/palaces (this represents patrician desire to be in the governor's good graces and in the more eminent neighbourhood):
        - visited by a governmental official from a larger governor's residence: small or medium positive;
        - visited by a governmental official from a "same dimension" governor's residence: very small or small positive;
        - visited by a governmental official from a smaller governor's residence: none or very small positive;
    - plebeian houses (governor's occasion to make propaganda counterbalanced by Housing inequalities factor):
        - visited by a governmental official from a 5x5 governor's residence: very small or small negative;
        - visited by a governmental official from a 4x4 governor's residence: none;
        - visited by a governmental official from a 3x3 governor's residence: very small or small positive.
- politics:
    - each patrician villa/palace pays (this represents governor's ability to oppose patrician power, which is stronger in absence of a Senate):
        - if visited by a governmental official from a larger governor's residence, 100%  of the potential owed taxes;
        - if visited by a governmental official from a "same dimension" governor's residence or if a larger governor's residence exists, only 85%  of the potential owed taxes, or 95% without a built Senate;
        - if visited by a governmental official from a smaller governor's residence or if a "same dimension" governor's residence exists, only 75%  of the potential owed taxes, or 92% without a built Senate.

**Additional (not so) random thoughts**:
- Senate and governor's residences could be monuments:
    - a Senate under construction should mitigate or eliminate negative sentiment factor affecting patrician villas/palaces;
    - they could cost a bit more.
- Senate could be as much desirable as governor's palace.
- Governor's residences could condition average housing level.
- Governor's residences could affect prosperity, as expressed in the paper manual.
- Governor's residences could be paid by governor's personal savings, as expressed in the paper manual
- Patrician villas/house should not require schools (rich children had private Greek tutors) and should not gain entertainment bonus from taverns (which were attended by plebeians).

(edit: two small additions to consider Governor's residence built without Senate, upgraded Senate and Senate desirability).
